css/style.scss

```diff
@@ -1095,3 +1095,63 @@ input[type="password"] {
 		filter: none !important;
 	}
 }
+
+.icon-videolarge {
```

Comment:
Use the following instead:
```css
@include icon-black-white('menu-people', 'spreed', 1);
```
This makes sure it works correctly in all available color modes of Nextcloud.
Afterwards add it to the exception block in:
https://github.com/nextcloud/spreed/blob/339b1dfc05e3ff01c89f58757febb7a7fa97e887/css/style.scss#L323-L326

To make sure it's always white in the call screen.